sw/tools/airframe_editor/xml_airframe.py

```diff
@@ -76,13 +92,18 @@ def reorganize_airframe_xml(airframe_xml):
     find_and_add_sections_with_name(airframe_xml, airframe, "XSENS")
 
     find_or_add_group(airframe_xml, airframe, "GAINS")
+
     # Fixedwing
-    find_and_add_sections_with_name(airframe_xml, airframe, "HORIZONTAL CONTROL")
+    find_and_add_sections_with_name(airframe_xml, airframe,
```

Comment:
IMHO we don't need to enforce a 80char limit... 100 or even 120 is just as fine nowadays and IMHO makes it more readable in cases like this.